lib/database/release.php

```diff
@@ -82,61 +91,16 @@ function isValidConsoleId($consoleId)
 function getEmulatorIdByConsoleId($consoleId)
```

Comment:
is this function even used? shouldn't it lookup the mappings from releases.php?

Reply:
It's used here https://github.com/RetroAchievements/RAWeb/blob/develop/public/dorequest.php#L185 to point to a specific emulator by console ID which is ambiguous by now - also might be ambiguous by looking it up via the releases file.
Ideally we can deprecate/remove it altogether but I'm not sure whether any `latestclient` RPC calls still use the `c` parameter instead of `e`.

Reply:
Ambiguous as in Console ID `1` would resolve to RALibretro, RetroArch and RAGens - but RAGens is the one that potentially identifies via console ID instead of emulator ID `0`.
We should have a look at usage data to determine which clients can be removed from that list.

Reply:
Makes sense. The DLL has been passing `e` since 0.75 (4 Feb 2019), and as this list only supports emulators that use the DLL and we require the user to upgrade the DLL, I think it's fine to remove the associated code.

Reply:
That's great to hear - I'll deprecate and remove it in a separate PR.

Reply:
See https://github.com/RetroAchievements/RAWeb/commit/2256225a828bd268b4f1d61439bb3dbd0d2e58b6